x/accounts/testing/account_abstraction/partial_account.go

```diff
@@ -0,0 +1,68 @@
+package account_abstraction
```

Comment:
implements the dummy account meant to test account abstraction + pub key rotation (pub key rotation is for spiciness)